.github/workflows/rebase-suite-e.yml

```diff
@@ -0,0 +1,45 @@
+name: Rebase the suite-e branch on develop
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Fetch the suite-e branch
+        run: git fetch origin suite-e:suite-e
+      - name: Rebase the suite-e branch on develop
```

Comment:
is it possible to reuse this? https://github.com/trezor/trezor-suite/blob/595960c775474d96fcf2588f2125b25254e71e2e/.github/workflows/bot-rebase.yml

Reply:
this particular script is for rebasing on comment as I understood it, I believe it might be better so see what's going on